### PR TITLE
ci: Update brew before installing clang-format

### PR DESCRIPTION
**Pull request by philipphofmann**
#skip-changelog

### CODE REVIEW

**Review by kevinrenskers — APPROVED** [review]

**Comment by kevinrenskers** [comment]
LGTM

File: .github/workflows/format-code.yml [hunk 1]
```diff
@@ -16,9 +16,10 @@ jobs:
     runs-on: macos-11
     steps:
       - uses: actions/checkout@v3
+      ## Update internal list of formulae to the latest
+      - run: brew update
       - name: Install Clang-Format
         run: brew install clang-format
-      - run: clang-format --version
```
Comment (philipphofmann) [inline .github/workflows/format-code.yml:21]: This is redundant. The install log above already displays the version.

Comment (kevinrenskers) [reply .github/workflows/format-code.yml:21]: True. But I already merged it, sorry 😬

Comment (kevinrenskers) [reply .github/workflows/format-code.yml:21]: Oh wait I actually didn't, weird. So you can still change the PR.

Comment (philipphofmann) [reply .github/workflows/format-code.yml:21]: No worries, I added it 😁

**Pull request merged**